test/remote/gateways/remote_ixopay_test.rb

```diff
@@ -14,6 +14,7 @@ def setup
       email: 'test@example.com',
       description: 'Store Purchase',
       ip: '192.168.1.1',
+      stored_credential: stored_credential(:recurring)
```

Comment:
Let me know if I'm wrong, but it looks like this method sets `network_transaction_id`. I _think_ you said something in a comment that Ixopay doesn't return a `network_transaction_id`, though; is that right? And does it affect if this test should be used like this?

Reply:
It looks to me like the helper method only supplies a `network_transaction_id` if `id` is specified: https://github.com/activemerchant/active_merchant/blob/master/test/test_helper.rb#L249

Am I reading that correctly?

Reply:
I think in this case, it'll end up setting `id` to `nil` on that line, and then assigning that value to `network_transaction_id` [here](https://github.com/activemerchant/active_merchant/blob/master/test/test_helper.rb#L252). Maybe that's okay? I'm honestly not sure, but figured it was worth pointing out.

Reply:
I think that a value of `nil` is fine. In truth, Core will send almost every GSF with a value of `nil` if a specific value isn't provided. We just don't want to send any sort of `network_transaction_id` to Ixopay or save a value for it within our vault (which is handled in Core).

Reply:
Ah, okie doke. Thanks for explaining that, then!